CONTRIBUTING.md

```diff
@@ -56,6 +56,39 @@ So this approach has been abandoned a while ago.
 
 ### Releasing LALRPOP
 
+#### Semver policy
+
+All releases should follow rust's semantic versioning guidelines, as described
+at https://doc.rust-lang.org/cargo/reference/semver.html. Breaking changes in
+our context include changes that break previously working lalrpop grammars.
```

Comment:
And breaking changes(though rare as they are) to the Configuration api used in build scripts?

Reply:
Yeah, my intent was to assume those were breaking and spell out the additional case that isn't breaking, but I'll push a change to this to make that more explicit in a moment here.